Make KeyCodes Ord (#2918)
rust-windowing/winit: CHANGELOG.md

# Unreleased

> - Add `PartialOrd` and `Ord` to `KeyCode` and `NativeKeyCode` to allow them to be compared, e.g. to store them into a `BTreeSet`

Reviewer: No need to have that many details.

```suggestion
- Implement `PartialOrd` and `Ord` for `KeyCode` and `NativeKeyCode`.
```
